playground/tests/e2e.sh

```diff
@@ -126,7 +126,7 @@ signer_init()
         ""                  # Configure root ? [enter to continue]
         ""                  # Configure targets? [enter to continue]
         "1"                 # Configure online roles? [1: configure key]
-        "LOCAL_TESTING_KEY" # Enter key id
+        "4"                 # Enter key id
```

Comment:
nit pick: this would make it easier to find the right place in the code
```suggestion
        "4"                 # Select online keytype
```

same nit applies to two other locations